### ISSUE TITLE
Feature request: commands at console

### ISSUE THREAD

**Issue by SlimeDog** [Ugleh/RedstoneProximitySensor#11]
RPS 2.4.4

Please support
- rps give ...
- rps list ...

at the console.

**Comment by Ugleh** [comment]
Should have already been naturally implemented. I'll check to see what it isn't.

**Comment by Ugleh** [comment]
So just tried it out and "rps give" and "rps list" work, at least on my 1.14.2 server. Are you sure it isn't working?

When you use rps list make sure you type out a players name, "rps list" by itself would only list the current players.

**Comment by SlimeDog** [comment]
```
>rps list
[12:34:18 INFO]: [RPS] Sorry, you must be in-game to use that command.
>rps list PLAYER_NAME_ELIDED
[12:34:11 INFO]: [RPS] Player not found.
```
So, expanded feature requests:
- **rps list PLAYER** from the console, even if PLAYER is not connected.
- **rps list PLAYER** in-game, even if PLAYER is not connected.
- Use strings from language.yml as entered (ie., without colors). I will add colors (or other formatting) if I want it.
- On a related matter, support setting the prefix [RPS] -- in language.yml (or config.yml), so that colors (or other formatting) may be changed. I use a blue background console, so red and magenta are very difficult to read.

**Comment by Ugleh** [comment]
There isn't a non-depricated way to get an offline players sensors by their name, so I am going to implement Mojangs API of grabbing a players UUID, which means to use /rps list [player]. and the player isn't online requires connection to the internet.

**Comment by SlimeDog** [comment]
Great! Even though I block most outbound network calls, I allow UUID look-ups from Mojang. Other plugins have similar needs.

**Comment by Ugleh** [comment]
I'll post an update tomorrow, have 1 more commit to make. Thanks.

**Comment by SlimeDog** [comment]
Confirmed fixed in RPS 2.4.5:
- **rps list PLAYER** from console
- **rps list PLAYER** with PLAYER not connected
- Formatting managed in language.yml

Thank you.
